docs/synapse/userguides/storm_ref_filter.ipynb

```diff
@@ -125,7 +125,7 @@
    "source": [
     "# Use previous temp cortex, define and print test query\n",
     "q = '<query> '\n",
-    "q1 = 'inet:fqdn'\n",
+    "q1 = 'inet:fqdn '\n",
```

Comment:
what's the purpose of this change?

Reply:
I commented on it in chat a while back. The resulting query before was `inet:fqdn-inet:fqdn=google.com` which we decided was invalid.